notebooks/ch14/bias_modeling.md

```diff
@@ -229,15 +230,28 @@ Cross-validation provides a more accurate method of estimating our model error u
 
-Underfitting is typically caused by too much bias; overfitting is typically caused by too much variance.
+Underfitting is typically caused by too much bias; overfitting is typically caused by too much model variance.
 
-Collecting more data reduces variance. One recent trend is to select a model with low bias and high intrinsic variance (e.g. a neural network) and collect many data points so that the model variance is low enough to make accurate predictions. While effective in practice, collecting enough data for these models tends to require large amounts of time and money.
+Collecting more data reduces variance. For example, the model variance of linear regression goes down by a factor of $ 1
+/n $, where $ n $ is the number of data points. Thus, doubling the dataset halves the model variance, and collecting many data points will cause the variance to approach 0. One rece end is to select a model with low bias and high intrinsic variance (e.g. a neural network) and collect many data points so that the model variance is low enough to make accurate predictions. While effective in practice, collecting enough data for these models tends to require large amounts of time and money.
 
 Collecting more data reduces bias if the model can fit the population process exactly. If the model is inherently incapable of modeling the population (as in the example above), even infinite data cannot get rid of model bias.
 
 Adding a useful feature to the data, such as a quadratic feature when the underlying process is quadratic, reduces bias. Adding a useless feature rarely increases bias.
 
-Adding a feature, whether useful or not, typically increases model variance. In order to increase a model's prediction accuracy, a new feature should decrease bias more than it increases variance.
+Adding a feature, whether useful or not, typically increases model variance. This does so because each new feature addas a a new model parameter, increasing the total number of models. In order to increase a model's prediction accuracy, a new feature should decrease bias more than it increases variance. 
+
+Removing features will typically increase bias and can cause underfitting. For example if the underlying data comes from a quadratic curve, removing the quadratic feature and only using a simple linear model won't capture the trend of the data. 
+
+The plot below is helpful in summarizing the takeaways mentioned above:
+
+![bias_modeling_bias_var_plot.png](https://raw.githubusercontent.com/DS-100/textbook/master/assets/bias_modeling_bias_var_plot.png)
+
+As model complexity increases, we will be able to predict better values for our training set, but not extrapolate well to the test set. By decreasing model complexity, the bias will be higher for the training set, but there will be less model variance.
+
+As we add more data, we shift our plot to the right and down, reducing bias and variance:
```

Comment:
We don't "shift a plot". Instead, we shift the curves on the plot.

> In the plot below, we observe that adding more data effectively moves the bias and variance curves down and to the right. This not only reduces both bias and variance but also allows us to use more complex models to achieve low test set error.

Reply:
will be fixed